ui/src/app/components/object-detail/object-detail.component.ts

```diff
@@ -399,39 +462,60 @@ export class ObjectDetailComponent implements OnInit {
         let oldRow = this.tableData[j]
         let standardDataType = pgSQLToStandardTypeTypemap.get(col.spDataType)
         if (col.spColMaxLength !== undefined && col.spColMaxLength !== 'MAX') {
-          if ((col.spDataType === 'STRING' || col.spDataType === 'VARCHAR') && typeof col.spColMaxLength === "number" && col.spColMaxLength > ColLength.StringMaxLength) {
+          if (
+            (col.spDataType === 'STRING' || col.spDataType === 'VARCHAR') &&
+            typeof col.spColMaxLength === 'number' &&
+            col.spColMaxLength > ColLength.StringMaxLength
+          ) {
             col.spColMaxLength = 'MAX'
-          } else if (col.spDataType === 'BYTES' && typeof col.spColMaxLength === "number" && col.spColMaxLength > ColLength.ByteMaxLength) {
+          } else if (
+            col.spDataType === 'BYTES' &&
+            typeof col.spColMaxLength === 'number' &&
+            col.spColMaxLength > ColLength.ByteMaxLength
+          ) {
             col.spColMaxLength = 'MAX'
           }
         }
-        if (typeof (col.spColMaxLength) === 'number') {
+        if (typeof col.spColMaxLength === 'number') {
           col.spColMaxLength = col.spColMaxLength.toString()
         }
-        if (col.spDataType != 'STRING' && col.spDataType != 'BYTES' && col.spDataType != 'VARCHAR') {
-          col.spColMaxLength = ""
+        if (
+          col.spDataType != 'STRING' &&
+          col.spDataType != 'BYTES' &&
+          col.spDataType != 'VARCHAR'
+        ) {
+          col.spColMaxLength = ''
         }
         if (col.srcId == this.tableData[j].srcId && this.tableData[j].srcId != '') {
           updateData.UpdateCols[this.tableData[j].srcId] = {
             Add: this.tableData[j].spId == '',
             Rename: oldRow.spColName !== col.spColName ? col.spColName : '',
             NotNull: col.spIsNotNull ? 'ADDED' : 'REMOVED',
             Removed: false,
-            ToType: (this.conv.SpDialect === Dialect.PostgreSQLDialect) ? (standardDataType === undefined ? col.spDataType : standardDataType) : col.spDataType,
+            ToType:
+              this.conv.SpDialect === Dialect.PostgreSQLDialect
+                ? standardDataType === undefined
```

Comment:
Please revert all the lint changes. Only show code changes done for the Check constraint feature